runtime/lua/vim/secure.lua

```diff
@@ -86,21 +110,77 @@ function M.read(path)
     trust[fullpath] = hash
   end
 
+  write_trust(trust)
+
+  return contents
+end
+
+--- Update the trust status of file at {path} in the trust database at
+--- $XDG_STATE_HOME/nvim/trust.
+---
+---@param path (string) Path to a file to update status for.
+---@param mode (string) One of the following:
+---   - 'allow': Add file to trust database and set it as trusted
+---   - 'deny': Add file to trust database and set it as denied
+---   - 'forget': Remove file from trust database
```

Comment:
In the Vim help syntax, words between single quotes are interpreted as links to options (e.g. `'textwidth'`). Use double quotes here to prevent that.

```suggestion
---   - "allow": Add file to trust database and set it as trusted
---   - "deny": Add file to trust database and set it as denied
---   - "forget": Remove file from trust database
```

Reply:
Resolved in 8bf34bd892d5362f0e79d04dd5cbb28fb303c9f1